### ISSUE TITLE
tslint warnings 

### ISSUE THREAD

**Issue by ry** [denoland/deno#912]
Currently when I run `./tools/lint.py` I get the following
```
> ./tools/lint.py
python /Users/rld/src/deno/third_party/cpplint/cpplint.py --filter=-build/include_subdir --repository=src --extensions=cc,h --recursive src/.
node /Users/rld/src/deno/third_party/node_modules/tslint/bin/tslint -p . --exclude **/gen/**/*.ts
node /Users/rld/src/deno/third_party/node_modules/tslint/bin/tslint ./js/**/*_test.ts ./tests/**/*.ts --exclude **/gen/**/*.ts
Warning: The 'no-circular-imports' rule requires type information.
Warning: The 'restrict-plus-operands' rule requires type information.
```
TODO resolve these warnings.

**Comment by DanielRamosAcosta** [comment]
This is already fixed by #916

**Comment by ry** [comment]
Thx
